packages/node/test/index.test.ts

```diff
@@ -177,16 +177,15 @@ describe('SentryNode', () => {
             expect(event.exception!.values![0]).not.toBeUndefined();
             expect(event.exception!.values![0].stacktrace!).not.toBeUndefined();
             expect(event.exception!.values![0].stacktrace!.frames![2]).not.toBeUndefined();
-            expect(event.exception!.values![0].stacktrace!.frames![2].pre_context).toBeUndefined();
-            expect(event.exception!.values![0].stacktrace!.frames![2].post_context).toBeUndefined();
+            expect(Array.isArray(event.exception!.values![0].stacktrace!.frames![2].pre_context)).toBe(true);
+            expect(Array.isArray(event.exception!.values![0].stacktrace!.frames![2].post_context)).toBe(true);
```

Comment:
This test now checks that context **is** included. 

I need to add a test that checks that no context is included when the integration is disabled...